src/main/java/io/github/syst3ms/skriptparser/lang/Variable.java

```diff
@@ -39,7 +39,7 @@ public Variable(VariableString name, boolean local, boolean list, Class<?> type)
         this.supertype = ClassUtils.getCommonSuperclass(this.type);
     }
 
-    private Object getRaw(TriggerContext ctx) {
+    public Object getRaw(TriggerContext ctx) {
```

Comment:
Ideally, variables would work in such a way that this hacky workaround be unnecessary, but that might have to wait.

Reply:
So it currently can stay in the code?

Reply:
Yeah